packages/react-native-bpk-component-calendar/src/android/build.gradle

```diff
@@ -18,21 +18,23 @@ def safeExtGet(prop, fallback) {
 
 def _reactNativeVersion = safeExtGet("reactNative", "+")
 def _compileSdkVersion = safeExtGet("compileSdkVersion", 28)
-def _backpackVersion = safeExtGet("backpackVersion", "47ae74e60fbeac32aa1e03375547eb5f6623cce5")
+def _targetSdkVersion = safeExtGet("targetSdkVersion", 28)
+def _minSdkVersion = safeExtGet("minSdkVersion", 21)
+def _buildToolsVersion = safeExtGet("buildToolsVersion", "28.0.3")
 
 android {
   compileSdkVersion _compileSdkVersion
-  buildToolsVersion safeExtGet("buildToolsVersion", "26.0.3")
+  buildToolsVersion _buildToolsVersion
 
   defaultConfig {
-    minSdkVersion safeExtGet("minSdkVersion", 21)
-    targetSdkVersion safeExtGet("targetSdkVersion", 28)
+    minSdkVersion _minSdkVersion
+    targetSdkVersion _targetSdkVersion
 
     versionName getVersionFromJsPackage(project)
   }
 }
 
 dependencies {
   compileOnly "com.facebook.react:react-native:${_reactNativeVersion}"
-  implementation "com.github.skyscanner:backpack-android:${_backpackVersion}"
+  implementation "com.github.skyscanner:backpack-android:2.5.0"
```

Comment:
Changed to using the minimum version supported. Maven will always get the latest version if we publish it like this, which is what we want. 

The version I published in our internal repo is already using this version (because the root project is using it too) so this doesn't really change anything. Is just more specific and safe than before.